bigflow/commons.py

```diff
@@ -232,7 +232,9 @@ def decode_version_number_from_file_name(file_path: Path):
 
 
 def get_docker_image_id(tag):
+    logger.info("Getting docker image ID.")
     images = subprocess.getoutput(f"docker images -q {tag}")
```

Comment:
btw `bigflow.commons.run_process` put stdout/stderr to logger (with debug or info).

Reply:
I have changed getoutput to run_process